staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -490,6 +494,20 @@ func NewKubectlCommand(o KubectlOptions) *cobra.Command {
 	// Stop warning about normalization of flags. That makes it possible to
 	// add the klog flags later.
 	cmds.SetGlobalNormalizationFunc(cliflag.WordSepNormalizeFunc)
+
+	var err error
+	o.Arguments, err = pref.ApplyAliases(cmds, o.Arguments, o.IOStreams.ErrOut)
+	if err != nil {
+		fmt.Fprintf(o.IOStreams.ErrOut, "error occurred while applying aliases %v\n", err)
+		os.Exit(1)
+	}
+	os.Args = o.Arguments
```

Comment:
For the reviewers; this is needed to append the arguments defined in kuberc.

Reply:
That seems very fishy, I'd expect you pass the necessary elements internally inside the cobra.Command, rather than modify os-injected args.

Reply:
Thank you for review. I agree with that. I'll look for alternatives to not modify the `os.Args`.

Reply:
I believe that I've found a way. If we call root command's `SetArgs` function, appended args will be carried to actual command and no need to modify `os.Args` anymore.